### DISCUSSION TITLE
Virtual character shape offset

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#77] [Q&A] [answered]
Virtual character has a shape offset `mShapeOffset`:
https://jrouwe.github.io/JoltPhysics/class_character_virtual_settings.html#a71029078959636aa6a6bca8ab72f31e3

If a shape is a decorated one and already specifies a position offset (e.g. `RotatedTranslatedShape` used in the example), would `mShapeOffset` add a local position offset on top of whatever `RotatedTranslatedShape` already specifies?

**Comment by jrouwe** [comment, marked as answer]
> would mShapeOffset add a local position offset on top of whatever RotatedTranslatedShape already specifies?

Yes it will.
